core/lib/src/response/content.rs

```diff
@@ -89,4 +89,5 @@ ctrs! {
     Plain: Plain, "plain text", "text/plain",
     Css: CSS, "CSS", "text/css",
     JavaScript: JavaScript, "JavaScript", "application/javascript"
+    Toml: Toml, "TOML", "application/toml"
```

Comment:
I'm pretty sure this should be `Toml: TOML, "TOML", "application/toml"` (second one uppercased to match the definition earlier). The `JavaScript` line above needs a trailing comma, as well.

Reply:
Good catch. I fixed that up, although I think I'm parsing your above comment as prefering not to include toml?

The underlying problem I have is purely that I need to send some toml in a response, so if there's a way to create content types at runtime I'm happy to do that as well, grepping around it seemed like this was the only way to do it.

Reply:
You can simply `ContentType::new("application", "json")`. If you're trying to set the content-type on a response, use [`Content`](https://api.rocket.rs/rocket/response/struct.Content.html).